rad/src/types/mod.rs

```diff
@@ -95,9 +100,48 @@ impl RadonTypes {
             RadonTypes::Float(inner) => inner,
             RadonTypes::Integer(inner) => inner,
             RadonTypes::Map(inner) => inner,
+            RadonTypes::RadonError(_) => unreachable!("`RadonTypes::RadonError` is not operable"),
```

Comment:
`unreachable!` is used to indicate that this branch cannot be entered at runtime, but the compiler is not smart enough to infer it. For example, if you have already checked that `self` is not a `RadonError`. [Rust docs have a nice example](https://doc.rust-lang.org/nightly/std/macro.unreachable.html#examples)

Since nothing is preventing users from calling `RadonTypes::RadonError().as_operable()`, then this should be a panic.

```suggestion
            RadonTypes::RadonError(_) => panic!("`RadonTypes::RadonError` is not operable"),
```